src/Microsoft.NetCore.Analyzers/UnitTests/Runtime/SpecifyIFormatProviderTests.cs

```diff
@@ -915,6 +915,240 @@ End Class
 ");
         }
 
+        [Fact]
+        public async Task CA1305_Parse_Diagnostic()
```

Comment:
Not really related to this change but I couldn't find any test with the parse methods and I wanted to have some comparison to see how should work the `TryParse` variants.